src/util/debug_module.js

```diff
@@ -41,7 +41,9 @@ util.inspect.defaultOptions.depth = 10;
 util.inspect.defaultOptions.colors = true;
 util.inspect.defaultOptions.breakLength = Infinity;
 
-
+//TODO : Need to finalize the solution for process.env, adding it to avoid system crash
+process.env.LOG_TO_SYSLOG_ENABLED = config.LOG_TO_SYSLOG_ENABLED ? 'true' : 'false';
+process.env.LOG_TO_STDERR_ENABLED = config.LOG_TO_STDERR_ENABLED ? 'true' : 'false';
```

Comment:
I think this should be in config.js so it will live reload -
https://github.com/noobaa/noobaa-core/blob/00513d49bc10c86f6cbd52e73d67fede40de8c7c/config.js#L1090-L1097

Reply:
should we also change the default in config.js to output to STDERR only unless overridden by runtime config?
https://github.com/noobaa/noobaa-core/blob/00513d49bc10c86f6cbd52e73d67fede40de8c7c/config.js#L475-L476

to:
```js
config.LOG_TO_SYSLOG_ENABLED = false;
config.LOG_TO_STDERR_ENABLED = true;

```